test/configtx.yaml

```diff
@@ -70,6 +70,9 @@ Organizations:
             Endorsement:
                 Type: Signature
                 Rule: "OR('Org1MSP.peer')"
+        AnchorPeers:
```

Comment:
@bestbeforetoday , I suppose we can set AnchorPeers during channel creation. But an error happens when commit chaincode as below:
```
msg: "failed to commit chaincode: rpc error: code = FailedPrecondition desc = no combination of peers can be derived which satisfy the endorsement policy: no peer combination can satisfy the endorsement policy",
```
is there any thing missed in channel config? or I will create another PR for adjust channel config.

Reply:
The MSPDir says the organization is `org1.example.com` so I think the first peer will be named `peer0.org1.example.com`.